### ISSUE TITLE
CoreUObject wrong function decleration.

### ISSUE THREAD

**Issue by marcocorriero** [CorrM/Unreal-Dumping-Agent#13]
```
template<typename T>
T* FindObject(const std::string& name);

template<typename T>
T* FindObject()

etc..
```

this function must be static.

**Comment by marcocorriero** [comment]
And
```
class UFunction : public UStruct
{
public:
	int32_t                                            FunctionFlags                                             ; // 0x0000(0x0000) NOT AUTO-GENERATED PROPERTY
	int16_t                                            RepOffset                                                 ; // 0x0000(0x0000) NOT AUTO-GENERATED PROPERTY
	int8_t                                             NumParms                                                  ; // 0x0000(0x0000) NOT AUTO-GENERATED PROPERTY
	unsigned char                                      pad_4M[1]                                                 ; // 0x0000(0x0000) NOT AUTO-GENERATED PROPERTY
	int16_t                                            ParmsSize                                                 ; // 0x0000(0x0000) NOT AUTO-GENERATED PROPERTY
	uint16_t_t                                         ReturnValueOffset                                         ; // 0x0000(0x0000) NOT AUTO-GENERATED PROPERTY
	uint16_t_t                                         RPCId                                                     ; // 0x0000(0x0000) NOT AUTO-GENERATED PROPERTY
	uint16_t_t                                         RPCResponseId                                             ; // 0x0000(0x0000) NOT AUTO-GENERATED PROPERTY
	class UProperty*                                   FirstPropertyToInit                                       ; // 0x0000(0x0000) NOT AUTO-GENERATED PROPERTY
	class UFunction*                                   EventGraphFunction                                        ; // 0x0000(0x0000) NOT AUTO-GENERATED PROPERTY
	int                                                EventGraphCallOffset                                      ; // 0x0000(0x0000) NOT AUTO-GENERATED PROPERTY
	unsigned char                                      pad_PC[4]                                                 ; // 0x0000(0x0000) NOT AUTO-GENERATED PROPERTY
	class *                                            Func                                                      ; // 0x0000(0x0000) NOT AUTO-GENERATED PROPERTY


	static UClass* StaticClass()
	{
		static auto ptr = UObject::FindClass("Class CoreUObject.Function");
		return ptr;
	}
};
```

UFunction Wrong Member Types

**Comment by 2217936322** [comment]
NOT AUTO-GENERATED PROPERTY

**Comment by marcocorriero** [comment]
> NOT AUTO-GENERATED PROPERTY

soo wut?

**Comment by 2217936322** [comment]
> > NOT AUTO-GENERATED PROPERTY
> 
> soo wut?
so you can fix

**Comment by marcocorriero** [comment]
> > > NOT AUTO-GENERATED PROPERTY
> > 
> > 
> > soo wut?
> > so you can fix

Lol, that was actually CorrM making this issue thru my pc.
Im his fav Bug-hunter.

**Comment by 2217936322** [comment]
> > > > NOT AUTO-GENERATED PROPERTY
> > > 
> > > 
> > > soo wut?
> > > so you can fix
> 
> Lol, that was actually CorrM making this issue thru my pc.
> Im his fav Bug-hunter.

o(￣▽￣)ｄ you are so good man

**Comment by CorrM** [comment]
NOT AUTO-GENERATED PROPERTY => mean i already know this class have this member so just i will not generate it i will just say that class have this member !!

**Comment by CorrM** [comment]
fixed 476f789a7618425aa04ed4dd9222c042974254e6
